### ISSUE TITLE
tikz latex template should use standalone option [tikz] for marginless tikz diagrams

### ISSUE THREAD

**Issue by PeterSommerlad** [pandoc-ext/diagram#15]
As of today the standalone document template for tikz diagrams looks like:

```lua
local tikz_template = pandoc.template.compile [[
\documentclass{standalone}
\usepackage{tikz}
$for(header-includes)$
$it$
$endfor$
$additional-packages$
\begin{document}
$body$
\end{document}
]]
```

this way all images get a left-hand border, even when cropped.

However, the latex package [standalone](https://texdoc.org/serve/standalone.pdf/0) has a dedicated `[tikz]` option that gives much cleaner results and renders images without a left-hand border. So the above template should be replaced with

```lua
--- LaTeX template used to compile TikZ images. Takes additional
--- packages as the first, and the actual TikZ code as the second
--- argument.
local tikz_template = pandoc.template.compile [[
\documentclass[tikz]{standalone}
$for(header-includes)$
$it$
$endfor$
$additional-packages$
\begin{document}
$body$
\end{document}
]]
```

**Comment by PeterSommerlad** [comment]
see pull request #16

**Comment by tarleb** [comment]
Fixed via PR #16.
